### DISCUSSION TITLE
Eigenvector Continuity or MAC to Categorize Modes?

### DISCUSSION THREAD

**Discussion by ovaisnajeeb** [petrobras/ross#1128] [Q&A]
How does ROSS identify what eigenvalue belongs to what mode as speed changes? Does it use the simple Eigenvalue Continuity method or does it use Modal Assurance Criteria?

**Comment by jguarato** [comment]
Hi @ovaisnajeeb,

In ROSS, these criteria have not yet been implemented. No technique is used to identify modes as speed changes.
